### DISCUSSION TITLE
Webpack 5 dealing with globals differently - Not found in changelog

### DISCUSSION THREAD

**Discussion by michael-whelan** [webpack/webpack#17996] [Q&A]
# Bug report
version: Webpack 5.89

**What is the current behavior?**
When clicking on some elements of the site we redirect to partner sites (in a new tab). 
The decision to redirect to a partner, to another area of the site or to block with a sign up request is made in a third party service. 
As a result when the response is returned to allow the redirect we use `globals.window.open(leadHref, "_blank")` to open the new tab.
Since the upgrade to webpack5 we noticed that the `globals.window.open(leadHref, "_blank")` was being blocked on ios mobile with the note popup blocked.

If any changes have happened to the way webpack deals with the `window.open` in version 5 I could not find them noted in the changelog.

**If the current behavior is a bug, please provide the steps to reproduce.**
Visit site on ios mobile
Click store element on feed.
Witness that the new tab did not open.
Click a second time -> tab opens on partner site as expected.

**What is the expected behavior?**
Clicking on an element and after the decision to open a partner site we should open a tab with the partner site.

**Other relevant information:**
webpack version:   5.89
Node.js version: >=18.x
Operating System: ios mobile
Additional tools:

**Comment by alexander-akait** [comment]
We don't change your globals. And we don't have different logic for iOS or any other browsers/os/etc. Please debug your code.

> If the current behavior is a bug, please provide the steps to reproduce.
> Visit site on ios mobile
> Click store element on feed.
> Witness that the new tab did not open.
> Click a second time -> tab opens on partner site as expected.

Sorry I can't help you using your steps
